src/tigerbeetle/main.zig

```diff
@@ -421,18 +452,45 @@ const Command = struct {
 
 fn replica_release_execute(replica: *Replica, release: vsr.Release) noreturn {
     assert(release.value != replica.release.value);
+    if (builtin.os.tag != .linux) @panic("replica_release_execute unsupported");
 
     for (replica.releases_bundled.const_slice()) |release_bundled| {
         if (release_bundled.value == release.value) break;
     } else {
-        log_main.err("{}: release_execute: release {} is not available; upgrade the binary", .{
+        log_main.err("{}: release_execute: release {} is not available; upgrade (or downgrade) the binary", .{
             replica.replica,
             release,
         });
-        @panic("release_execute: binary missing required version");
+        @panic("release not available");
+    }
+
+    const multiversion = replica.multiversion.?;
```

Comment:
Great idea! It might still need the tag check and panic for now, due to platform specific differences (same as `if (multiversion != null and builtin.target.os.tag == .linux) multiversion.?.start();` above)